gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/MetricNames.java

```diff
@@ -22,6 +22,8 @@
  */
 public class MetricNames {
 
+  public static final String NUM_WORKUNITS = "numWorkUnits";
```

Comment:
Should this be grouped into SourceMetrics and have the metric name as "gobblin.source.numWorkUnits.created" to be consistent with other metric names?

Reply:
Are you suggesting a new class SourceMetrics ?